.eslintrc.js

```diff
@@ -35,8 +35,8 @@ module.exports = {
         objects: 'always-multiline',
         imports: 'always-multiline',
         exports: 'always-multiline',
-        functions: 'ignore'
-      }
-    ]
-  }
+        functions: 'ignore',
+      },
+    ],
+  },
```

Comment:
I owe this file a couple small additions. Ref https://github.com/roots/bud/blob/8a06e499a3d209dfb54a50b4aabb4500a3a67455/packages/sage/src/sage/eslint.ts#L4-L48

Reply:
Done.